src/v2/controllers/FormController.ts

```diff
@@ -238,6 +239,16 @@ export default Controller.extend({
     // Run hook: transform the user name (a.k.a identifier)
     const values = this.transformIdentifier(formName, model);
 
+    // widget rememberMe feature stores the entered identifier in a cookie, to pre-fill the form on subsequent visits to page
+    if (this.options.settings.get('features.rememberMe') && this.options.settings.get('features.rememberMyUsernameOnOIE')) {
```

Comment:
is that logical operator right, i.e., what happens if either is falsey?
why do we have both? if reasons are historical, can we do something about consolidating them? the `OnOIE` suffix feels like implementation detail leakage.

Reply:
Hisortical reasons:

- https://github.com/okta/okta-signin-widget/pull/3002/files#diff-9d2ede01662be7d3291032cd313089a2f2a7399c19f8f078a7b2910c2a38f141L22
- https://github.com/okta/okta-signin-widget/pull/3002/files#diff-9d2ede01662be7d3291032cd313089a2f2a7399c19f8f078a7b2910c2a38f141L40

Per conversation with PM, they want to combine the FFs into 1, however they want that to be a separate change